modules/nw-aws-local-zones-user-workload-ingress.adoc

```diff
@@ -0,0 +1,9 @@
+:content-type: CONCEPT
```

Comment:
same comment for file modules/installation-aws-local-zones-ref-deployment.adoc

Reply:
Marking this for removal.